src/Rule/BlankLineAfterDirective.php

```diff
@@ -48,15 +47,13 @@ public function check(Lines $lines, int $number): ?string
         }
 
         $lines->next();
+        $nextLine = $lines->current();
 
-        if (RstParser::isOption($lines->current())) {
```

Comment:
this line previously throws a type error as `Lines::current()` returns null instead of `Line` ( see: https://3v4l.org/MQjsU ).